starter-code/apple-pie-starter-code/index.html

```diff
@@ -2,34 +2,51 @@
 <html lang="en">
 <head>
   <title>Apple Pie Recipe</title>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" type="text/css" media="screen" href="./css/main.css" />
+  <script src="main.js"></script>
 </head>
-<body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <header><div class = "presentation1"><div class="image1"> <img src = "../apple-pie-starter-code/images/apple-pie.jpg" alt="Apple pie" style="width:100%;"></div>
+    <div class="h1-centered"><h1>Apple Pie</h1></div>
+    <div class= "presentation2"><p>This was my grandmother's apple pie recipe. I have never seen another one
+  quite like it. It will always be my favorite and has won me several first place
+  prizes in local competitions. I hope it becomes one of your favorites as well!</p></div>
+</header>
+  <body>
+  <div class="image-recipe"><img src="images/recipe-info.png"></div>
+  </div>
 
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
+  <hr width= 300px>
 
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
-flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
-temperature and let simmer.
+  <div class = "ingredients"><h2>Ingredients</h2>
```

Comment:
You don't have to wrap your h2 in a div, if you want to add specific styling to this element just add a class to the h2, because to many may lead to some weird results (box in box in box..)